test/FunctionalTests/Infrastructure/ExecutionContextLoggingProvider.cs

```diff
@@ -77,11 +77,15 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                 // if it is written in the test's execution context.
                 ExecutionContext.Run(_executionContext, s =>
                 {
-                    var timestamp = $"{(DateTimeOffset.UtcNow - _timeStart).TotalSeconds.ToString("N3")}s";
+                    var timestamp = $"{(DateTimeOffset.UtcNow - _timeStart).TotalSeconds:N3}s";
```

Comment:
Is this a new syntax for specifying the format? I've never seen this before.

Reply:
It's always been possible